addons/udes_priorities/models/priority.py

```diff
@@ -37,6 +45,12 @@ def _default_sequence(self):
         string="Stock Transfer Types",
         help="The stock picking types this priority is visible on. No picking types means all picking types",
     )
+    affected_pickings = fields.Many2many(
```

Comment:
We can get rid of this field. It's only important for the user to know if there are any outstanding pickings when they try to change a priority. If they want to find the actual pickings then they can manually search themselves in the picking view.